homeassistant/components/person/__init__.py

```diff
@@ -423,6 +421,8 @@ def __init__(self, config: dict[str, Any]) -> None:
         self._gps_accuracy: float | None = None
         self._source: str | None = None
         self._unsub_track_device: Callable[[], None] | None = None
+        self._attr_state: str | None = None
```

Comment:
Just to clarify, state is not a mutable object so this attribute can be initialized either as a class attribute or as a instance attribute.

Reply:
Ok, wasn't sure about this. Thanks 👍